scenarios/kubernetes_verify.py

```diff
@@ -29,10 +29,10 @@
 VERSION_TAG = {
     '1.11': '1.11-v20190318-2ac98e338',
     '1.12': '1.12-v20190318-2ac98e338',
-    '1.13': '1.13-v20190318-2ac98e338',
-    '1.14': '1.14-v20190318-2ac98e338',
+    '1.13': '1.13-v20190817-cc05229',
+    '1.14': '1.14-v20190817-cc05229',
     # this is master, feature branches...
-    'default': '1.14-v20190318-2ac98e338',
+    'default': '1.14-v20190817-cc05229',
```

Comment:
this is just for consistency. for > 1.14 we're not using this. we can delete this script when 1.14 goes out of support